### DISCUSSION TITLE
Where do I put the firmware for the enhancement chips?

### DISCUSSION THREAD

**Discussion by possiblyneal** [Marat-Tanalin/bsnes-mt#29] [Q&A] [answered]
Where do I put the firmware? or Does the MT branch not use it or include it?

**Comment by Marat-Tanalin** [comment, marked as answer]
bsnes-mt is based on bsnes, so it supports all bsnes features, and if not mentioned on the bsnes-mt page, in the same way as original bsnes. So just put firmware files to the same folder you would with regular bsnes.

**Reply by possiblyneal** [reply]
Thank you!
